src/lib/y2packager/dialogs/addon_selector.rb

```diff
@@ -147,17 +202,35 @@ def continue_msg
       # description widget
       # @return [Yast::Term] the addon details widget
       def details_widget
-        MinHeight(3,
-          VWeight(25, RichText(Id(:details), Opt(:disabled), "<small>" +
-          description + "</small>")))
+        VWeight(40, RichText(Id(:details), Opt(:disabled), "<small>" +
+        description + "</small>"))
       end
 
       # extra help text
       # @return [String] translated text
       def description
         # TRANSLATORS: inline help text displayed below the product selection widget
-        _("The dependencies between products are not handled automatically. " \
-          "The dependent modules or extensions must be selected manually.")
+        _("Select a product to see it's description here. The dependent products " \
+          "are selected automatically.")
+      end
+
+      def product_description(product)
+        erb_file = File.join(__dir__, "product_summary.erb")
+        log.info "Loading ERB template #{erb_file}"
+        erb = ERB.new(File.read(erb_file))
+
+        # compute the dependant products
+        dependencies = []
+        if product.depends_on && !product.depends_on.empty?
+          product.depends_on.each do |p|
```

Comment:
so with new rubocop merged (not yet), you can use `product.depends_on&.each` and avoid condition completelly.

Reply:
OK, that will be changed by the Rubocop automatically (or at least detected). So let's fix that later...